src/CTA.Rules.Models/IDEFileActions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Codelyzer.Analysis.Model;
+
+namespace CTA.Rules.Models
+{
+    public class IDEFileActions
```

Comment:
Could you add a doc comment for the class that describes its use and how it differs from other file actions?

Also, would it make sense to rename anything with the prefix `IDE` to something more generic? Dependencies shouldn't be aware of the libraries or projects that consume them, but I also don't have full context here about how this model is consumed.

Reply:
We don't usually add doc comments for models. We can document our models classes at a later stage if needed.